llvm/lib/Target/WebAssembly/WebAssemblyISelLowering.cpp

```diff
@@ -1830,11 +1816,8 @@ SDValue WebAssemblyTargetLowering::LowerLoad(SDValue Op,
 
     SDValue Idx = DAG.getTargetConstant(*Local, Base, MVT::i32);
     EVT LocalVT = LN->getValueType(0);
-    SDValue LocalGet = DAG.getNode(WebAssemblyISD::LOCAL_GET, DL, LocalVT,
-                                   {LN->getChain(), Idx});
-    SDValue Result = DAG.getMergeValues({LocalGet, LN->getChain()}, DL);
-    assert(Result->getNumValues() == 2 && "Loads must carry a chain!");
-    return Result;
+    return DAG.getNode(WebAssemblyISD::LOCAL_GET, DL, {LocalVT, MVT::Other},
```

Comment:
Why has this changed?

Reply:
The original code fails verification enabled by this change.
`LOCAL_GET` is declared with `SDNPHasChain` property, and thus should have both a chain operand and a chain result. The code on the left creates a node without a chain result, which causes a check in `SDNodeInfo::verifyNode()` to fail.

Reply:
Ah, yes, thanks.